platforms/native/engine/jsb-spine-skeleton.js

```diff
@@ -290,12 +290,17 @@ const cacheManager = require('./jsb-cache-manager');
         }
     };
 
-    skeleton.setSkeletonData = function (skeletonData) {
+    skeleton._updateUITransform = function () {
+        const skeletonData = this._skeletonData;
```

Comment:
this._runtimeData is not assigned on native platform

Reply:
But why isn't there a problem in v3.8.5 ?

Reply:
SkeletonData setted by editor, can not trigger set property of skeletonData. In the file of Skeleton.ts, _updateUITransform will set the content size to 100, 100